src/binding/petsc4py/makefile

```diff
@@ -79,9 +79,11 @@ rst2html: checkdocutils
 
 SPHINXBUILD = sphinx-build
 SPHINXOPTS  =
-.PHONY: sphinx sphinx-html sphinx-pdf sphinx-man sphinx-info
+.PHONY: sphinx sphinx-html sphinx-pdf sphinx-man sphinx-info sphinx-apidoc
 sphinx: sphinx-html sphinx-pdf sphinx-man sphinx-info
-sphinx-html:
+sphinx-apidoc:
+	sphinx-apidoc src/petsc4py/ -o docs/source/ -f -T
```

Comment:
`sphinx-apidoc` is not automatic enough. I prefer to use the `autosummary` extension, that's how I generate mpi4py API reference.

Reply:
Sure. I'll take this out